platform-core-module/src/main/java/com/softicar/platform/core/module/event/SystemEventConfirmAction.java

```diff
@@ -0,0 +1,61 @@
+package com.softicar.platform.core.module.event;
+
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.common.io.resource.IResource;
+import com.softicar.platform.core.module.CoreI18n;
+import com.softicar.platform.core.module.CoreRoles;
+import com.softicar.platform.dom.elements.DomElementsImages;
+import com.softicar.platform.emf.action.AbstractEmfButtonAction;
+import com.softicar.platform.emf.action.IEmfManagementAction;
+import com.softicar.platform.emf.authorization.role.IEmfRole;
+import com.softicar.platform.emf.form.IEmfFormBody;
+import com.softicar.platform.emf.predicate.IEmfPredicate;
+import java.util.Optional;
+import java.util.function.Supplier;
+
+public class SystemEventConfirmAction extends AbstractEmfButtonAction<AGSystemEvent> implements IEmfManagementAction<AGSystemEvent> {
```

Comment:
Well this heavily depends on the assumed workflow. My idea is that the system administrator just confirms that they have seen the problem, and took action, will take actions, of just ignore the notification. Don't forget this can be an error message or a warning.